tests/datadriven/test_variant_annotations.py

```diff
@@ -0,0 +1,255 @@
+"""
+Data-driven tests for variants.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import glob
+import json
+
+import vcf
+
+import ga4gh.datamodel as datamodel
+import ga4gh.datamodel.datasets as datasets
+import ga4gh.datamodel.variants as variants
+import ga4gh.datarepo as datarepo
+import ga4gh.protocol as protocol
+import tests.datadriven as datadriven
+
+
+def testVariantAnnotationSets():
+    testDataDir = "tests/data/datasets/dataset1/variants"
+    for test in datadriven.makeTests(testDataDir, VariantAnnotationSetTest):
+        yield test
+
+
+class VariantAnnotationSetTest(datadriven.DataDrivenTest):
+    """
+    Data driven test class for variant annotation sets. Builds an
+    alternative model of a variant annotation set using pyvcf, and
+    verifies that it is consistent with the model built by the
+    variants.VariantAnnotationSet object.
+    """
+    def __init__(self, variantAnnotationSetId, baseDir):
+        self._dataset = datasets.AbstractDataset("ds")
+        self.datarepo = datarepo.FileSystemDataRepository("tests/data")
```

Comment:
We should be consistent with private/public stuff here. Either make everything public for these test classes or make everything protected. Mixing the two  is confusing (a reader will look for meaning here: _why_ is this one public and the others are protected?).